test/integration/render/scenarios/variables/setter/scenario.js

```diff
@@ -0,0 +1,16 @@
+const map = new carto.Map({
+    container: 'map',
+    background: 'black'
+});
+
+const source = new carto.source.GeoJSON(sources['points']);
+// Check that precision is good
+const viz = new carto.Viz(`
+    @oneHundred: 10
+    width: ramp(linear($numeric, 0, 10), [0.10,0.20,0.30]) * @oneHundred
+`);
+const layer = new carto.Layer('layer', source, viz);
+
+layer.addTo(map);
+
+viz.variables.oneHundred = 100;
```

Comment:
missing newline, you can configure this with [editorconfig](http://editorconfig.org/)